lib/snapshot.go

```diff
@@ -598,6 +645,7 @@ func (snap *Snapshot) ProcessSnapshotChunk(mainDb *badger.DB, mainDbMutex *deadl
 	})
 }
 
+// TODO: should we still do checksum stuff before we hit the first snapshot height?
```

Comment:
Hmm... I think we do unfortunately. Since we don't want to "stop the world" and compute the checksum when we compute the snapshot height. Hopefully it's OK though?